app/assets/stylesheets/app/typography.scss

```diff
@@ -1,5 +1,5 @@
-a {
-  text-decoration: underline;
+a { /* "links" */
+  text-decoration: none;
```

Comment:
I go back and forth where I usually like my links underlined so it's obvious what's clickable. Especially for people who might not be able to distinguish color as well. It's less of a thing in listings and menu bars. Definitely within a sentence/paragraph I think it's critical. Maybe we can add `p a` is underlined so that the link is underlined if it's within a paragraph?